adaptive_n_back/experiment.js

```diff
@@ -177,12 +180,14 @@ var instruction_node = {
 var update_delay_block = {
 	type: 'call-function',
 	func: update_delay,
+	data: {trial_id: "update delay"},
```

Comment:
what are these ids for? generally variables should not have spaces like this, but underscore "_"